core/sail/memory/src/test/java/org/eclipse/rdf4j/sail/memory/benchmark/QueryBenchmark.java

```diff
@@ -68,6 +72,16 @@
 
 	List<Statement> statementList;
 
+	public static void main(String[] args) throws RunnerException {
```

Comment:
I also noticed that you added main methods to each file. I'll have to try with IntelliJ to see if it doesn't mess up the plugins detection of what to run. I'm away from my computer for a week now, so just merge it and I'll give it a go sometime later. ￼

Reply:
No rush, I'm supposed to be on holiday myself (at home, we're still in lockdown here).